app/helpers/ems_infra_helper/textual_summary.rb

```diff
@@ -175,4 +177,11 @@ def textual_host_default_vnc_port_range
     value = "#{@ems.host_default_vnc_port_start} - #{@ems.host_default_vnc_port_end}"
     {:label => _("%{title} Default VNC Port Range") % {:title => title_for_host}, :value => value}
   end
+
+  def textual_topology
+    {:label => _('Topology'),
+     :image => 'topology',
+     :link  => url_for(:controller => 'infra_topology', :action => 'show', :id => @ems.id),
```

Comment:
change this to '/infra_topology' (with the forward slash) to fix the other test error